src/components/PuzzleModal/index.scss

```diff
@@ -0,0 +1,67 @@
+@use "./stylesheets/responsiveness";
+@use "./stylesheets/typography";
+
+.puzzlemodal__container {
+  position: absolute;
+  margin-left: auto;
+  margin-right: auto;
+  left: 0;
```

Comment:
<img width="379" alt="pop-up-puzzle-text" src="https://user-images.githubusercontent.com/38510322/147315642-fe3827af-c85e-479e-8134-1405652df835.PNG">

Can we decrease the margin/padding here?

Reply:
I could decrease the padding on the inner elements, but I use the margin properties there to center it in the screen